### ISSUE TITLE
CLI application gets hang.

### ISSUE THREAD

**Issue by r0hi7** [r0hi7/DockerENT#29]

**Comment by r0hi7** [comment]
This issue was observed in MacOS where the system throwing this warning below.
```
objc[26309]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
objc[26309]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
objc[26310]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
objc[26310]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
```
This is through macos Catalina update this has been discussed in detail @ https://stackoverflow.com/questions/50168647/multiprocessing-causes-python-to-crash-and-gives-an-error-may-have-been-in-progr

This issues has nothing to do with application `DockerENT`

**Comment by r0hi7** [comment]
TL;DR stack overflow: `export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES`
